src/main/java/com/incognia/common/ReasonCode.java

```diff
@@ -0,0 +1,64 @@
+package com.incognia.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum ReasonCode {
+  ADDRESS_CITY_COUNT_30D("evidence_address_city_count_30d"),
```

Comment:
I just noticed that you use the listing reasons instead of the assessment reasons. [This](https://www.notion.so/inlocoglobal/Expose-assessment-reasons-API-only-30f75f255a154e79afe359a1b878e2c9#923d0d4ff211456485df737471c2ecfc) is the correct list of reasons (you can also look at `AssessmentReasonCode`, excluding `UNKNOWN`.

Reply:
good call, i'll change it soon

Reply:
done